spec/workers/process_content_change_and_generate_emails_worker_spec.rb

```diff
@@ -209,6 +210,9 @@
 
       expect(Email.count).to eq(100)
       expect(Email.all.pluck(:address).uniq.count).to eq(100)
+      DatabaseCleaner.allow_remote_database_url = true
+      DatabaseCleaner.strategy = :truncation
+      DatabaseCleaner.clean
```

Comment:
We need this to happen after the test, regardless of what happens during the test. While having this code at the end was sufficient for testing, if the test crashes before the end, this cleanup won't happen.

Is there some way of telling RSpec to do this after this test?